src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyoutViewModel.cs

```diff
@@ -228,7 +229,7 @@ public bool Submit()
             }
 
             SmartRenameViewModel?.Commit(IdentifierText);
-            Session.Commit();
+            _ = Session.CommitAsync(previewChanges: false).ReportNonFatalErrorAsync();
```

Comment:
what is tracking this async work?  for example, integratoin tests will need to know once CommitAsync is done.  we need IAsyncToken tracking here.

Reply:
token added.